easybuild/easyconfigs/v/VASP/VASP-5.4.4-intel-2017a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '5.4.4'
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale
+materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics,
+from first principles."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+# Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
+sources = ['%s.%s.tar.gz' % (name.lower(), version)]
+
+dependencies = [
+    ('FFTW', '3.3.6'),
+    ('zlib', '1.2.11', '', True),
+]
+
+prebuildopts = 'cp arch/makefile.include.linux_intel ./makefile.include && '
+
+# path to libfftw3xf_intel.a is hardcoded in makefile.include
+prebuildopts += 'sed -i "s|\$(MKLROOT)/interfaces/fftw3xf|\$(FFTW_LIB_DIR)|" makefile.include && '
+
+# remove mkl flag to prevent mixing dynamic libs with the static libs in LIBBLACS/SCALAPACK
+prebuildopts += 'sed -i "s|-mkl=sequential||" makefile.include && '
```

Comment:
Makefile.include now contains -mkl=sequential. If not removed, VASP reports Can not load symbol MKLMPI_Get_wrappers